transport/api/v1/editor.go

```diff
@@ -20,10 +20,10 @@ type verifyData struct {
 }
 
 func (h *Handler) Login(w http.ResponseWriter, r *http.Request) {
-	var transportEditor apiEditor.Editor
+	var transportEditor transportModel.Editor
 	err := json.NewDecoder(r.Body).Decode(&transportEditor)
 	if err != nil {
-		util.WriteResponse(w, http.StatusBadRequest, "Error reading request body")
+		util.WriteResponse(w, http.StatusBadRequest, "Error reading request body: "+err.Error())
```

Comment:
Mohli bychom nějak ten magic link udělat tak, že na strance po kliknuti linku uvidíme access token? I mean not from url, but from page, it's very hard to extract access token from url

Reply:
Bohužel neni jak, ten token je za fragmentem (hashem - http://localhost:8080/api/v1#access_token=eyJ...), takže ho na BE nepřečteme, museli bychom mít dummy FE jenom pro tenhle usecase

Reply:
muzeme v admince zmenit redirect page na napriklad hostname/api/v1/login_successful? Vytvorime endpoint, ktery potom vrati v response body neco jako "Your access token: 'sometoken', refresh token: 'somerefreshtoken'"

Ten endpoint by mohl vratit jenom takovy string a status http 200.

Vidim, ze v magic linku v emailu mame redirect url https://pmjfifolpefrbsysbqro.supabase.co/auth/v1/verify?token=9128321893&type=magiclink&redirect_to=**http://localhost:8080/api/v1**

Reply:
Ale ten odkaz už je jednou redirectlej a říkám ti, že nemáme jak na BE ten token přečíst z URL, takže ho nemůžeme vrátit